pages/_app/index.jsx

```diff
@@ -28,11 +28,14 @@ class App extends NextApp {
     isAuthorized: false,
   }
 
+  store = initStore()
+
   handleRouteChange = (url) => {
     logPageView(url)
     const { dataProvider, activity } = new Route(url)
 
-    if (dataProvider) {
+    // eslint-disable-next-line no-restricted-globals
+    if (!isNaN(dataProvider)) {
```

Comment:
What the reason to use global instead Number?

If you ever have to disable linter, always add a comment before with the explanation of the reason for doing so.

Reply:
Number.isNaN doesn't convert its argument to Number

> true if the given value is NaN and its type is Number; otherwise, false.

I will change it to Number.isNaN(Number(dataProvider))